plugins/autocertkit

```diff
@@ -1632,9 +1632,18 @@ def _get_local_device_bridge(session, device):
             log.debug("%s is a bridge." % device)
             return device
 
-    raise Exception("Error: Cannot determine bridge name of given device name %s." % device)
+    log.debug("Cannot determine bridge of given device %s." % device)
 
-    return devname
+    if device.startswith('xenbr'):
+        log.debug("Try using device name %s." % device)
+        return device
+
+    elif device.startswith('eth'):
```

Comment:
> Newly created network can have different name than xenbrX from XS 7.0

Doesn't this mean that it's _not_ safe to assume all ethX's have a corresponding xenbrX?

I agree that if we reach this far, we need to do a local call to get that info, but that's not going to be the most expensive operation. The important thing is to fail at the right place (i.e. we can't figure out what the bridge actually should be).

Does that make sense?